### ISSUE TITLE
go-get & go-config

### ISSUE THREAD

**Issue by lloiser** [jnmoal/go-guru#9]
Just wanted to inform you that both [go-config](https://github.com/joefitzgerald/go-config) and [go-get](https://github.com/joefitzgerald/go-get) have been consolidated into [go-plus](https://github.com/joefitzgerald/go-plus) as of go-plus [v5.0.0](https://github.com/joefitzgerald/go-plus/releases/tag/v5.0.0). See https://github.com/joefitzgerald/go-plus/pull/503. This means that both go-get and go-config will be unpublished from atom soon so installing your package in atom will fail after that.

In order to fix this you have to change your code so that it will install go-plus instead of go-get and go-config (see [package-deps in package.json](https://github.com/jnmoal/go-guru/blob/7b15dc246bc9eedc9b4239f941e0a74f75a3e2e2/package.json#L24-L27)). Beside that nothing else changes because go-plus provides the same services as go-get and go-config now do. (see PR https://github.com/jnmoal/go-guru/pull/10)

Yet there is another opportunity for you: integrate go-guru into go-plus!
* it will be part of a way bigger community so everyone can benefit from your awesome package 😉 
* more maintainers so we can share knowledge and issues could be fixed faster
* go-plus provides some other nice features like a built-in panel which you could use to display your results instead of having an additional panel.

This can either be done by you or if you want we (the maintainers of go-plus) can do this. There is already a simple integration of guru in go-plus anyway so this won't be a big deal.

/cc @joefitzgerald

**Comment by pwaller** [comment]
Ping. Every time I open my editor I get messages about failed installations of go-config and go-get. I'd love to see this fixed :)

**Comment by sporkmonger** [comment]
Yeah, second the ping. This issue is starting to move beyond "nuisance" territory since it happens with every new Atom window. And also because I actually use this package all the time.

**Comment by jnmoal** [comment]
Hello, I'm really sorry for the delayed reponse.

I'd like to integrate this package with go-plus.
How can I achieve this?

Thank you for everything.

**Comment by lloiser** [comment]
Depends: do you want to move your code over to go-plus or let your code here and simply use what is provided by go-plus.

If you want to move your code:

Start by reading a bit through the code at [`go-plus`](https://github.com/joefitzgerald/go-plus). As you can see everything integrated there is structured in separate folders.

You also would need a separate tab in the go-plus panel at the bottom of atom. Just have have a look at any existing view component like the [`godoc`](https://github.com/joefitzgerald/go-plus/tree/master/lib/doc) integration.

[`lib/godef.js`](https://github.com/joefitzgerald/go-plus/blob/master/lib/navigator/godef.js#L76-L135) might be worth a look as it already uses `guru` to "navigate" through go code (`guru definition`) .

If you feel comfortable, simply checkout go-plus, make changes and open a PR.

**Comment by jnmoal** [comment]
Ok.

I think I'll move the code over to go-plus.

I'll work on it this week end (might have some time).

Thanks!

**Comment by zamicol** [comment]
I'm seeing this issue too.  

For the search engines:

Failed to install go-guru dependencies
These packages were not installed, check your console
for more info.
  • go-get
  • go-config

![image](https://cloud.githubusercontent.com/assets/5232965/22272216/05a85b30-e256-11e6-82a9-8e6a1bb831fe.png)

**Comment by gburanov** [comment]
same for me. Is there solution that work already now?

**Comment by joefitzgerald** [comment]
We've added support for a subset of Guru directly into `go-plus`. We can extend this support further, and I would encourage you to file issues there for desired functionality. We would like to focus on individual commands, rather than generically supporting all of Guru, as I think it offers a better user experience.

**Comment by arquio** [comment]
○ → apm install go-get
Installing go-get to /home/$USER/.atom/packages ✗
Request for package information failed: Not Found

○ → apm install go-config
Installing go-config to /home/$USER/.atom/packages ✗
Request for package information failed: Not Found
